src/main/java/org/apache/commons/lang3/Functions.java

```diff
@@ -491,9 +491,27 @@ public static void tryWithResources(FailableRunnable<? extends Throwable> pActio
     }
 
     /**
-     * Rethrows a {@link Throwable} as an unchecked exception.
-     * @param pThrowable The throwable to rethrow
-     * @return Never returns anything, this method never terminates normally
+     * <p>Rethrow a {@link Throwable} as an unchecked exception. If the argument is
```

Comment:
In general, Javadocs always start with "Gets", not "Get", "Returns", not "Return", and so on; so here let's keep "Rethrows", and not change to "Rethrow" SVP.